### ISSUE TITLE
Windows arm64 builds

### ISSUE THREAD

**Issue by wezm** [dprint/dprint#886]
Attempting to install a project that depends on `dprint` on a Windows ARM device results in this error:

```
npm error path D:\Source\TypeScript\node_modules\dprint
npm error command failed
npm error command C:\Windows\system32\cmd.exe /d /s /c node ./install.js
npm error node:internal/modules/cjs/loader:1222
npm error   throw err;
npm error   ^
npm error
npm error Error: Cannot find module '@dprint/win32-arm64/package.json'
npm error Require stack:
npm error - D:\Source\TypeScript\node_modules\dprint\install_api.js
npm error - D:\Source\TypeScript\node_modules\dprint\install.js
npm error     at Module._resolveFilename (node:internal/modules/cjs/loader:1219:15)
npm error     at Function.resolve (node:internal/modules/helpers:133:19)
npm error     at Object.runInstall (D:\Source\TypeScript\node_modules\dprint\install_api.js:23:52)
npm error     at Object.<anonymous> (D:\Source\TypeScript\node_modules\dprint\install.js:4:26)
npm error     at Module._compile (node:internal/modules/cjs/loader:1467:14)
npm error     at Module._extensions..js (node:internal/modules/cjs/loader:1551:10)
npm error     at Module.load (node:internal/modules/cjs/loader:1282:32)
npm error     at Module._load (node:internal/modules/cjs/loader:1098:12)
npm error     at TracingChannel.traceSync (node:diagnostics_channel:315:14)
npm error     at wrapModuleLoad (node:internal/modules/cjs/loader:215:24) {
npm error   code: 'MODULE_NOT_FOUND',
npm error   requireStack: [
npm error     'D:\\Source\\TypeScript\\node_modules\\dprint\\install_api.js',
npm error     'D:\\Source\\TypeScript\\node_modules\\dprint\\install.js'
npm error   ]
npm error }
npm error
npm error Node.js v22.4.1
```

With more ARM Windows machines entering the market recently (such as the Lenovo 7x I'm using) it would be great if there was a Windows arm64 build available.

**Comment by wezm** [comment]
Wow that was quick. Thank you!

**Comment by dsherret** [comment]
That's cool you got one of the arm windows laptops. I updated the npm build to use the x64 release. Should be released now in 0.47.2. Can you let me know if it works?

I opened https://github.com/dprint/dprint/issues/888 for a native arm64 build because it doesn't work atm.

**Comment by wezm** [comment]
> Can you let me know if it works?

![image](https://github.com/dprint/dprint/assets/21787/6a43ae45-5854-4570-b23b-54dd3239d782)

Confirming it works. Screenshot shows running dprint 0.47.2 with arm64 node.js and x86-64 dprint binary.
